src/modules/company/repository/company-repository.ts

```
@@ -128,20 +143,24 @@
 
   async updatePassword(id, password: string): Promise<CompaniesEntity> {
-    const company = await this.companyRepository.findOneBy({id}).catch(handleError);
+    const company = await this.companyRepository
+      .findOneBy({ id })
+      .catch(handleError);
     const data = {
       recoverPasswordToken: null,
       password,
     };
 
     delete company.password;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)